cinema/serializers.py

```diff
@@ -112,14 +112,13 @@ class Meta:
 
 class TicketSerializer(serializers.ModelSerializer):
     def validate(self, attrs):
```

Comment:
The TicketSerializer validate method should catch the ValidationError and handle it properly. Right now, if the ticket.clean() method raises a ValidationError, it will break the flow of the application.